lab/lab1/part2-fizzbuzz.html

```diff
@@ -4,6 +4,27 @@
   </head>
   <body>
 
+    <h1>FizzBuzz custom</h1>
+
+    <form id="fbform">
+      <p>
+        Print numbers in the range
```

Comment:
this is fun 👍 

![image](https://user-images.githubusercontent.com/8103418/52544681-2f0dad00-2d80-11e9-920c-3a2f075a7c7b.png)